uui/components/layout/FlexItems/Panel.tsx

```diff
@@ -3,11 +3,21 @@ import { VPanel } from '@epam/uui-components';
 import css from './Panel.module.scss';
 
 interface PanelMods {
+    /*
```

Comment:
Rework according boolean guidelines - https://github.com/epam/UUI/wiki/Documentation-Guidelines#api-design-and-documentation-guidelines

Reply:
fixed